projects/plugins/jetpack/modules/masterbar/admin-menu/class-domain-only-admin-menu.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Domain-only sites Admin Menu file.
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Dashboard_Customizations;
+
+require_once __DIR__ . '/class-base-admin-menu.php';
+
+/**
+ * Class Domain_Only_Admin_Menu.
+ */
+class Domain_Only_Admin_Menu extends Base_Admin_Menu {
+
+	/**
+	 * Whether to use wp-admin pages rather than Calypso.
+	 *
+	 * Options:
+	 * false - Calypso (Default).
+	 * true  - wp-admin.
+	 *
+	 * @return bool
+	 */
+	public function should_link_to_wp_admin() {
+		return false;
+	}
```

Comment:
Do we need this one?

Reply:
It's an abstract method of the base class that needs to be implemented.